src/OpenTelemetry.Exporter.Console/ConsoleLogRecordExporter.cs

```diff
@@ -57,18 +57,18 @@ public override ExportResult Export(in Batch<LogRecord> batch)
                 else if (logRecord.StateValues != null)
                 {
                     this.WriteLine("LogRecord.StateValues (Key:Value):");
-                    for (int i = 0; i < logRecord.StateValues.Count; i++)
+                    foreach (var value in logRecord.StateValues)
```

Comment:
FYI this `foreach` allocates (an enumerator) where the previous `for` doesn't require anything.

Reply:
@alanwest would you address this?
(I can merge this now, and get a release process going. This can be added in a follow up PR)